src/generic_core/social.ts

```diff
@@ -626,6 +626,32 @@ export function notifyUI(networkName :string, userId :string) {
       });
     }
 
+    public processCloudNetworkData(networkData :any) :string {
+      if ((typeof networkData) === 'string') {
```

Comment:
This is weird. Don't you know the type, even if it's just an otherwise-untyped  `Object`?